bruker2nifti/metadata.py

```diff
@@ -0,0 +1,183 @@
+"""
+Provides the BrukerMetadata class for reading Bruker MRI metadata.
+
+The BrukerMetadata class should be instantiated for each study. Dictionaries
+are structured as follows:
+
+subject_data {
+  [key_from_subject_file]: [corresponding_value]
+}
+
+scan_data {
+  [scan_number]: {
+    'acqp': { [key_from_acqp_file]: [corresponding_value] },
+    'method': { [key_from_method_file]: [corresponding_value] },
+    'recons': {
+      [recon_number]: {
+        'reco': { [key_from_reco_file]: [corresponding_value] },
+        'visu_pars': { [key_from_visu_pars_file]: [corresponding_value] }
+      }
+    }
+  }
+}
+
+parse_subject() creates the subject_data dictionary and stores it in the object
+parse_scans() creates the scan_data dictionary and stores it in the object
+
+read_subject() returns a subject_data dictionary
+read_scans() returns a scan_data dictionary
+read_scan() returns the sub-dictionary corresponding to [scan_number]
+read_recons() returns the sub-citionary corresponding to 'recons' for a given
+  [scan_number]
+read_recon() returns the sub-dictionary corresponding to [recon_number] for a
+  given [scan_number]
+
+list_scans() returns a list of scan numbers
+list_recons() returns a list of recon numbers for a given scan
+"""
+import os
+
+from . import _utils as utils
```

Comment:
As above please use absolute paths respect to the package root

Reply:
Updated